ee/api/test/test_dashboard.py

```diff
@@ -36,9 +32,7 @@ def test_retrieve_dashboard_allowed_for_project_member(self):
         self.organization_membership.level = OrganizationMembership.Level.MEMBER
         self.organization_membership.save()
         ExplicitTeamMembership.objects.create(team=self.team, parent_membership=self.organization_membership)
-        dashboard = Dashboard.objects.create(
-            team=self.team, name="private dashboard", created_by=self.user, tags=["deprecated"]
-        )
```

Comment:
Setting tags with the Django orm won't work now. But also the reason I'm removing this instead of replacing with new tag initialization is because tags shouldn't work in non-ee tests anyways. This case is tested for in `posthog/api/test/test_tagged_item.py`.